src/components/Base/BaseToggleGroup.vue

```diff
@@ -1,18 +1,38 @@
 <template>
-  <ToggleGroupRoot v-model="modelValue" class="ToggleGroup" :type :disabled>
+  <ToggleGroupRoot
+    :modelValue
+    class="ToggleGroup"
+    :type="rekaType"
+    :disabled
+    @update:modelValue="
+      (val) => {
+        if (props.type === 'optionalSingle' || val !== undefined) {
```

Comment:
意図としてはこっちのほうがあってるかも？
（`type == "multiple"`のことを考えると）

```suggestion
        if (props.type !== "single" || val !== undefined) {
```